vNext/src/NuGet/Microsoft.Orleans.OrleansSQLUtils.nuspec

```diff
@@ -3,7 +3,7 @@
   <metadata>
     <id>Microsoft.Orleans.OrleansSqlUtils</id>
     <version>$version$</version>
-    <title>icrosoft Orleans Sql Utilities</title>
+    <title>Microsoft Orleans Sql Utilities</title>
```

Comment:
Look at this in src ;-) but these nuspecs will be removed in vNext, so don't have to bother with it, just "accidentally" fixed this typo.